src/classes/settingResolver.js

```diff
@@ -82,6 +82,10 @@ class SettingResolver extends Resolver {
     return command.help.name;
   }
 
+  async object(data) {
+    return data;
```

Comment:
Because every method from SettingResolver must return a Promise with a content, this is called **consistent return** when using pure abstraction. There's no JSDoc because, as you might have noticed, this PR has a **WIP** tag.